README.md

```diff
@@ -19,9 +19,9 @@ blend4j in larger open source projects include [Molgenis integration][molgensis]
 
 blend4j requires Java 8+ to function.
 
-[api]: http://jmchilton.github.io/blend4j/apidocs/
+[api]: http://galaxyproject.github.io/blend4j/apidocs/
 [tutorial]: https://github.com/apetkau/blend4j-tutorials
-[tests]: https://github.com/jmchilton/blend4j/tree/master/src/test/java/com/github/jmchilton/blend4j/galaxy
+[tests]: https://github.com/galaxyproject/blend4j/tree/master/src/test/java/com/github/galaxyproject/blend4j/galaxy
```

Comment:
This changes a package name from `com/github/jmchilton` to `com/github/galaxyproject`.  Can you keep as `jmchilton`?